libs/components/src/lib/tooltip/tooltip.directive.ts

```diff
@@ -95,7 +94,7 @@ export class TooltipDirective implements OnDestroy, OnInit {
     this.ariaDescriber.removeDescription(this.elementRef.nativeElement, this._message)
 
     // If the message is not a string (e.g. number), convert it to a string and trim it.
-    this._message = value != null ? `${value}`.trim() : ''
+    this._message = value !== null ? `${value}`.trim() : ''
```

Comment:
Does not the same anymore == null has true also for undefined values but === is strictly checking null